readme/WindowsBuild.md

```diff
@@ -1,82 +1,78 @@
-# Xournal++ Windows Build
+# Windows Build
 
-![Screenshot](main-win.png?raw=true "Xournal++ Screenshot on Win10")
+![Screenshot](./main-win.png?raw=true "Xournal++ Screenshot on Win10")
+
+### Install Dependencies
+
+Xournall requires the following software to build:
 
-## Preparation
 1. Install [MSYS2](https://www.msys2.org/) to a short path without spaces.
 2. Install [NSIS](https://nsis.sourceforge.io/Download) to the standard directory.
 3. Start Mingw-w64 64bit. (Always check if it says **MINGW64** - not 32bit and not MSYS2)
 
-This will open a console. All following steps happen in this console.
+This will open a console. The following steps happen in this console:
```

Comment:
Not sure this is an improvement; I would make this sentence more explicit:
```suggestion
This will open the MINGW64 console. All remaining steps in this document happen in the console.
```